### DISCUSSION TITLE
antd有考虑把所有的window，document改成根据node的ownerDocument | window来绑定事件吗

### DISCUSSION THREAD

**Discussion by apades** [ant-design/ant-design#50396] [General]
因为现在出了documentPictureInPicture这个API，通常都是在主window上init app然后把app root的node转移到docPIP中，就导致了绑定在window，document的事件没跟着去docPIP所在的window上。

像[material-ui](https://github.com/mui/material-ui/blob/3308f46d3811ae75127142d3d2dd89c963f962a7/packages/mui-utils/src/ownerWindow/ownerWindow.ts)都走这个获取window，就不会有这种绑定问题。

<del>我之前看了下内部用的rc-slider，好像要都改还挺麻烦的</del>

**Comment by afc163** [comment]
确实应该这么做，欢迎来 PR。
